pvanalytics/tests/quality/test_irradiance.py

```diff
@@ -42,28 +42,29 @@ def irradiance_qcrad():
     output = pd.DataFrame(
         columns=['ghi', 'dhi', 'dni', 'solar_zenith', 'dni_extra',
                  'ghi_limit_flag', 'dhi_limit_flag', 'dni_limit_flag',
+                 'ghi_extreme_limit_flag', 'dhi_extreme_limit_flag',
+                 'dni_extreme_limit_flag',
                  'consistent_components', 'diffuse_ratio_limit',
                  'consistent_components_outside_domain',
                  'diffuse_ratio_limit_outside_domain',
                  ],
-        data=np.array([[-100, 100, 100, 30, 1370, 0, 1, 1, 0, 0, 0, 1],
-                       [100, -100, 100, 30, 1370, 1, 0, 1, 0, 0, 1, 0],
-                       [100, 100, -100, 30, 1370, 1, 1, 0, 0, 1, 1, 1],
-                       [1000, 100, 900, 0, 1370, 1, 1, 1, 1, 1, 1, 1],
-                       [1000, 200, 800, 15, 1370, 1, 1, 1, 1, 1, 1, 1],
-                       [1000, 200, 800, 60, 1370, 0, 1, 1, 0, 1, 0, 1],
-                       [1000, 300, 850, 80, 1370, 0, 0, 1, 0, 1, 0, 1],
-                       [1000, 500, 800, 90, 1370, 0, 0, 1, 0, 1, 0, 1],
-                       [500, 100, 1100, 0, 1370, 1, 1, 1, 0, 1, 0, 1],
-                       [1000, 300, 1200, 0, 1370, 1, 1, 1, 0, 1, 0, 1],
-                       [500, 600, 100, 60, 1370, 1, 1, 1, 0, 0, 0, 0],
-                       [500, 600, 400, 80, 1370, 0, 0, 1, 0, 0, 0, 0],
-                       [500, 500, 300, 80, 1370, 0, 0, 1, 1, 1, 1, 1],
-                       [0, 0, 0, 93, 1370, 1, 1, 1, 0, 0, 1, 1],
-                       [100, 100, 0, 95, 1370, 0, 0, 1, 0, 0, 1, 1],
```

Comment:
Was it intentional that the last row got dropped?

Reply:
Nope, I'll add it back